core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.spec.ts

```diff
@@ -260,23 +261,24 @@ describe('Topics and skills dashboard page component', () => {
     componentInstance.filterBoxIsShown = false;
     componentInstance.toggleFilterBox();
     expect(componentInstance.filterBoxIsShown).toBeTrue();
+    componentInstance.toggleFilterBox();
+    expect(componentInstance.filterBoxIsShown).toBeFalse();
   });
 
   it('should display filter box on maximizing the window', () => {
```

Comment:
Ditto for this test too.

Reply:
@gp201 it was accidentally written now i have corrected this.